include/GloryDuckTables2019Lkl.h

```diff
@@ -0,0 +1,74 @@
+//////////////////////////////////////////////////////////////////////
+// Binned flux likelihood
+//////////////////////////////////////////////////////////////////////
+
+#ifndef GLORYDUCKLKL
+#define GLORYDUCKLKL
+
+#include "TObjArray.h"
+#include "TCanvas.h"
+
+#include "Lkl.h"
+
+class GloryDuckTables2019Lkl : public Lkl
+{
+ public:
+  // constructors
+  GloryDuckTables2019Lkl(TString inputString="");
+  
+  // destructor
+  virtual ~GloryDuckTables2019Lkl();
+ 
+  // make the TObjArray own its contents
+  void SetOwner(Bool_t set=kTRUE)            {fSampleArray->SetOwner(set);}
+
+  // add samples
+  inline void AddSample(Lkl* sample)        {fSampleArray->Add(sample); SetChecked(kFALSE);}
+
+  // getters
+  inline const TObjArray* GetSampleArray()     const {return fSampleArray;}
+  inline       Int_t      GetNSamples()        const {return fSampleArray->GetEntries();}
+  inline       Lkl*       GetReferenceSample() const {return (Lkl*)fSampleArray->At(0);}
+  inline       Lkl*       GetSample(Int_t i)   const {return (Lkl*)fSampleArray->At(i);}
+  inline       Double_t   GetLogJ()            const {return fLogJ;}
+  inline       UInt_t     GetNMasses()         const {return fNMasses;}
+  inline       UInt_t     GetActiveMass()      const {return fActiveMass;}
+
+  // unable SetUnitsOfG, dPhi/dE_signal must have proper units
+  virtual void SetUnitsOfG(Double_t unit);
+
+  // set mass index to the current active one
+  void SetActiveMass(Double_t mass);
```

Comment:
This function should return an Int_t (0 in case of success, 1 otherwise -this is actually a general rule for functions that don't need to return anything and that can fail depending on input etc)